airsonic-main/src/test/java/org/airsonic/player/dao/DaoTestCaseBean2.java

```diff
@@ -1,34 +1,23 @@
 package org.airsonic.player.dao;
 
 import org.airsonic.player.util.HomeRule;
+import org.airsonic.player.util.MigrationConstantsRule;
 import org.junit.ClassRule;
-import org.junit.Rule;
-import org.junit.runner.Description;
-import org.junit.runners.model.Statement;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.jdbc.core.JdbcTemplate;
-import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.junit4.rules.SpringClassRule;
-import org.springframework.test.context.junit4.rules.SpringMethodRule;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
 
-@ContextConfiguration(locations = {
-        "/applicationContext-service.xml",
-        "/applicationContext-cache.xml",
-        "/applicationContext-testdb.xml",
-        "/applicationContext-mockSonos.xml"})
```

Comment:
ContextConfiguration no longer required in SpringBoot 2 to set up contexts. It'll autodetect the configuration with SpringBootTest annotation